[2023-06-11T11:13:48.318+0000] {taskinstance.py:1097} INFO - Dependencies not met for <TaskInstance: first_dag_02.first_task scheduled__2021-11-01T00:00:00+00:00 [up_for_retry]>, dependency 'Not In Retry Period' FAILED: Task is not ready for retry yet but will be retried automatically. Current date is 2023-06-11T11:13:48.312252+00:00 and task will be retried at 2023-06-11T11:14:53.913785+00:00.
[2023-06-11T11:13:48.831+0000] {local_task_job_runner.py:154} INFO - Task is not able to be run
[2023-06-11T11:15:05.645+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_dag_02.first_task scheduled__2021-11-01T00:00:00+00:00 [queued]>
[2023-06-11T11:15:05.881+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_dag_02.first_task scheduled__2021-11-01T00:00:00+00:00 [queued]>
[2023-06-11T11:15:05.882+0000] {taskinstance.py:1308} INFO - Starting attempt 2 of 6
[2023-06-11T11:15:06.619+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): first_task> on 2021-11-01 00:00:00+00:00
[2023-06-11T11:15:06.745+0000] {standard_task_runner.py:57} INFO - Started process 267 to run task
[2023-06-11T11:15:06.765+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'first_dag_02', 'first_task', 'scheduled__2021-11-01T00:00:00+00:00', '--job-id', '538', '--raw', '--subdir', 'DAGS_FOLDER/First_dag.py', '--cfg-path', '/tmp/tmplmuhqtib']
[2023-06-11T11:15:06.767+0000] {standard_task_runner.py:85} INFO - Job 538: Subtask first_task
[2023-06-11T11:15:07.501+0000] {task_command.py:410} INFO - Running <TaskInstance: first_dag_02.first_task scheduled__2021-11-01T00:00:00+00:00 [running]> on host 7586ce0070ac
[2023-06-11T11:15:09.864+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Raja' AIRFLOW_CTX_DAG_ID='first_dag_02' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2021-11-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-11-01T00:00:00+00:00'
[2023-06-11T11:15:09.869+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-06-11T11:15:09.882+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo hello world, this is the first task!']
[2023-06-11T11:15:10.073+0000] {subprocess.py:86} INFO - Output:
[2023-06-11T11:15:10.145+0000] {subprocess.py:93} INFO - hello world, this is the first task!
[2023-06-11T11:15:10.154+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-06-11T11:15:10.555+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=first_dag_02, task_id=first_task, execution_date=20211101T000000, start_date=20230611T111505, end_date=20230611T111510
[2023-06-11T11:15:10.939+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-06-11T11:15:11.760+0000] {taskinstance.py:2651} INFO - 2 downstream tasks scheduled from follow-on schedule check
